airflow/models/dagrun.py

```diff
@@ -210,7 +234,7 @@ def __init__(
         execution_date: datetime | None = None,
         start_date: datetime | None = None,
         external_trigger: bool | None = None,
-        conf: Any | None = None,
+        _conf: Any | None = None,
```

Comment:
I don’t think it is necessary to rename this.

Reply:
@uranusjr Fixed